project/Common.scala

```diff
@@ -10,15 +10,16 @@ object Common extends AutoPlugin {
 
   override lazy val projectSettings =
     Dependencies.Common ++ Seq(
-    organization := "com.lightbend.akka.management", // FIXME proposing this change, then we can have "com.lightbend.akka.management|discovery" etc; we'd do management.commercial for example later
+    organization := "com.lightbend.akka.management",
     organizationName := "Lightbend Inc.",
+    organizationHomepage := Some(url("https://www.lightbend.com/")),
     startYear := Some(2017),
-    homepage := Some(url("https://github.com/akka/akka-management")),
+    homepage := Some(url("https://akka.io/")),
```

Comment:
or to the root of the akka-management docs? Either :+1: by me

Reply:
We point to `akka.io` in other Akka modules.